cli/cmd/root.go

```diff
@@ -88,7 +87,10 @@ func NewCmdRoot() *cobra.Command {
 
 			mw := io.MultiWriter(writers...)
 			log.Logger = zerolog.New(mw).Level(zerologLevel).With().Str("module", "cli").Timestamp().Logger()
-			if sentryDsn != "" && Version != "development" {
+
+			noTelemetry = getEnvOrDefault("CQ_NO_TELEMETRY", "false") == "true" || noTelemetry
+			sentryEnabled := sentryDsn != "" && Version != "development" && !noTelemetry
```

Comment:
I think this is incorrect now.

we have two different things which are unrelated:
telemetry - we send anonymised usage data such as source, destination plugins and number of resources used.
sentry (error-reporting) - we send panics that inccur in our official plugins and/or cli

those two should be either seperate and/or as suggested before have one flag `telemetry` with `none,errors,all` both should work but if someone disable analytics they shouldn't automatically disable error reporting (but they still should have the option). Same as in vscode and any other software.

Reply:
Ah, ok so I thought we don't have telemetry and https://github.com/cloudquery/cloudquery/pull/2619 introduces it.
Maybe it's worth just closing this PR and doing everything in https://github.com/cloudquery/cloudquery/pull/2619?

Not sure we need the resolution of `none,errors,all`, but best to leave for https://github.com/cloudquery/cloudquery/pull/2619

Reply:
Sounds good. let's close this one.